### DISCUSSION TITLE
Turn off "Best" Snapshot

### DISCUSSION THREAD

**Discussion by jsbowling42** [blakeblackshear/frigate#5253] [Ask A Question] [answered]
Is there a way to disable the "Best" snapshot feature? i would like the snapshot to be the image that caused the detection.

**Comment by NickM-27** [comment, marked as answer]
Probably just set the `best_image_timeout` to an incredibly high number

**Reply by jsbowling42** [reply]
wouldn't it still update the thumbnail if the current thumbnail has a better score than the previous?

**Reply by NickM-27** [reply]
I think so, it would reduce it though. There's currently no direct way to set the behavior you're asking for
